fidelisojeah/Hello-Books
# #152213424 User Borrow Book

#### What does this PR do?

- Allows User Borrow Book from the Library

#### How should this be manually tested?

- Create Account
- Verify Account
- Attempt to borrow book from library

#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/152213424
